### ISSUE TITLE
[Feat]  Scrape a code for an Invite only script on tradingview?

### ISSUE THREAD

**Issue by masterkanin** [mendableai/firecrawl#995]
CAn this scrap a code for an Invite only script on tradingview?

**Comment by rokbenko** [comment]
Hi, @masterkanin! I want to make sure I understand your question correctly. Are you asking if this project can scrape code from an invite-only script on TradingView? If that's the case, could you please provide more details or clarify your request?

**Comment by masterkanin** [comment]
Yes, Rokbenko! That’s exactly what I’m looking for. I want to use this strategy for personal purposes (https://www.tradingview.com/script/0z7HNhmV-QuantVue-Qkronos/). Could you let me know if it’s possible to scrape the code so I can use it personally?

**Comment by aatifprime** [comment]
can you please let me know if this can do this?

**Comment by masterkanin** [comment]
> can you please let me know if this can do this?

Yes i will!

**Comment by aatifprime** [comment]
@masterkanin can you telegram me @BTCblood

**Comment by masterkanin** [comment]
> Hi, @masterkanin! I want to make sure I understand your question correctly. Are you asking if this project can scrape code from an invite-only script on TradingView? If that's the case, could you please provide more details or clarify your request?

Any update

**Comment by ynrao24** [comment]
> > can you please let me know if this can do this?
> 
> Yes i will!

Can you please let me know if you can do this

**Comment by mogery** [comment]
Hi! Do you have a user account with access to the script?

**Comment by masterkanin** [comment]
> Hi! Do you have a user account with access to the script?

Yeah I do have an user account with access to the script

**Comment by 88Lom** [comment]
> Hi! Do you have a user account with access to the script?

Can we get the code?

**Comment by mogery** [comment]
We do not have any custom handling for this site, but you may be successful following the general instructions for Authentication:

We do not support authentication via providing a username/email and password, but there is a workaround if you're using the API or one of our SDKs (not on the Playground):

1. Sign into the target site from your browser
2. After signing in, open your browser's developer tools, go to the Network tab, and refresh the page.
3. Scroll to the top in the list of network requests, and click on the very first one.
4. In the Headers tab on the right (should be the default), scroll down to the Request Headers section, and copy the contents of your Cookie header.
5. Provide the Cookie header to the scraper in the scrape options like so:

```json
{
  "url": "https://example.com",
  "headers": {
    "Cookie": "PUT YOUR COOKIE HEADER VALUE HERE"
  }
}
```

Or, if you're doing a crawl:

```json
{
  "url": "https://example.com",
  "scrapeOptions": {
    "headers": {
      "Cookie": "PUT YOUR COOKIE HEADER VALUE HERE"
    }
  }
}
```

Adding better support for authentication is on our roadmap and we're working our way there.

**Comment by 88Lom** [comment]
> We do not have any custom handling for this site, but you may be successful following the general instructions for Authentication:
> 
> We do not support authentication via providing a username/email and password, but there is a workaround if you're using the API or one of our SDKs (not on the Playground):
> 
> 1. Sign into the target site from your browser
> 2. After signing in, open your browser's developer tools, go to the Network tab, and refresh the page.
> 3. Scroll to the top in the list of network requests, and click on the very first one.
> 4. In the Headers tab on the right (should be the default), scroll down to the Request Headers section, and copy the contents of your Cookie header.
> 5. Provide the Cookie header to the scraper in the scrape options like so:
> 
> {
>   "url": "https://example.com",
>   "headers": {
>     "Cookie": "PUT YOUR COOKIE HEADER VALUE HERE"
>   }
> }
> Or, if you're doing a crawl:
> 
> {
>   "url": "https://example.com",
>   "scrapeOptions": {
>     "headers": {
>       "Cookie": "PUT YOUR COOKIE HEADER VALUE HERE"
>     }
>   }
> }
> Adding better support for authentication is on our roadmap and we're working our way there.

where do we get the scraper?

**Comment by ViswaSuhaasPenugonda** [comment]
So is there a way to scrape the source code of a invite-only script on tradingview?
